frontend/src/components/pages/RobotModeler/ModelerSidebar/modelerSidebarFunctionality/modelerSidebarFunctionality.js

```diff
@@ -64,31 +66,24 @@ const modelerElementChangeHandler = (event, elementState, setterObject) => {
 };
 
 /**
- * @description Checks if tasks for selected application are already stored in session storage.
- * Otherwise, fetch tasklist from MongoDB.
+ * @description Sets all tasks for currently selected application from session storage
  * @param {String} application Application for which to get the tasks for.
- * @param {Object} setterObject object containing the functions for setting the state in the React component
+ * @param {Object[]} taskApplicationCombinations Array of task and application combination objects.
```

Comment:
Shouldn't the type be Array?

Reply:
Also fixed. i wasn't shure how to write "Array of objects" but I think just a simple `Array` is right.